FMB/FiniteModelBuilder.cpp

```diff
@@ -789,11 +794,13 @@ void FiniteModelBuilder::init()
   _distinctSortConstantCount.ensure(_sortedSignature->distinctSorts);
   _fminbound.ensure(env.signature->functions());
   for(unsigned f=0;f<env.signature->functions();f++){
+    if(env.signature->isTypeConOrSup(f)){ continue; }
     if(del_f[f]) continue;
 
     if(env.signature->functionArity(f)==0){ 
-      unsigned vsrt = env.signature->getFunction(f)->fnType()->result();
-      if(vsrt != Sorts::SRT_BOOL){
```

Comment:
I see `vsrt != Sorts::SRT_BOOL` on the left vs `SortHelper::isBoolSort(vsrtT)` on the right. Could this be inadvertently flipping the logic of the condition?

Reply:
It would indeed appear to be an inverting of the logic. I have now fixed.